src/Utils.php

```diff
@@ -107,17 +107,7 @@ public static function chooseHandler(): callable
      */
     public static function defaultUserAgent(): string
     {
-        static $defaultAgent = '';
-
-        if (!$defaultAgent) {
-            $defaultAgent = 'GuzzleHttp/Guzzle';
-            if (\extension_loaded('curl') && \function_exists('curl_version')) {
-                $defaultAgent .= ' curl/' . \curl_version()['version'];
-            }
-            $defaultAgent .= ' PHP/' . PHP_VERSION;
-        }
-
-        return $defaultAgent;
+        return 'GuzzleHttp';
```

Comment:
`return sprintf('GuzzleHttp/%d', ClientInterface::MAJOR_VERSION);`?

Reply:
Thank you